packages/core/src/builder.ts

```diff
@@ -59,26 +75,16 @@ function isCapabilityLookupCapableState(obj: unknown): obj is CapabilityLookupCa
  */
 export class Builder<State extends Partial<BuildableState>> {
```

Comment:
Why not
```suggestion
export const MkBuilder = (plugins: Plugins) => class Builder<State extends Partial<BuildableState>> {
```
?

And then there's no need for a `private plugins: Plugins` member.

Reply:
The `Mk-` based naming here I'm not too sure nor am I attached to it. It may be simpler to just have it be `builder` or `Builder` or similar.